bd-logger/src/test/logger_integration.rs

```diff
@@ -1767,24 +1792,8 @@ fn stats_upload() {
   setup.flush_and_upload_stats();
   let stat_upload = StatsRequestHelper::new(setup.server.next_stat_upload().unwrap());
   assert_eq!(
-    stat_upload.get_counter("logger:logs_received", labels! {"log_level" => "trace"}),
-    Some(1),
-  );
-  assert_eq!(
-    stat_upload.get_counter("logger:logs_received", labels! {"log_level" => "debug"}),
-    Some(1),
-  );
-  assert_eq!(
-    stat_upload.get_counter("logger:logs_received", labels! {"log_level" => "info"}),
-    Some(2),
-  );
-  assert_eq!(
-    stat_upload.get_counter("logger:logs_received", labels! {"log_level" => "warn"}),
-    Some(1),
-  );
-  assert_eq!(
-    stat_upload.get_counter("logger:logs_received", labels! {"log_level" => "error"}),
-    Some(2),
+    stat_upload.get_counter("logger:logs_received", labels! {}),
```

Comment:
not sure we need this one either? logs by log levels is even a instant insights default workflow

Reply:
OK I figured it might be useful just to see what is coming in. I can remove this also.